src/java/org/apache/cassandra/io/compress/BufferType.java

```diff
@@ -18,26 +18,43 @@
 package org.apache.cassandra.io.compress;
 
 import java.nio.ByteBuffer;
+import org.apache.cassandra.io.util.DirectIOUtils;
 
 public enum BufferType
 {
     ON_HEAP
     {
+        @Override
```

Comment:
Avoid redundant @Override annotations, convention of Apache Cassandra Code Style:
http://cassandra.apache.org/doc/latest/development/code_style.html

Reply:
Fixed.